src/reynir_correct/errtokenizer.py

```diff
@@ -1412,10 +1462,12 @@ def fix_compound_words(
                     span=2,
                 )
             )
+            t1.origin_spans = pspan
```

Comment:
Consider passing pspan as a `origin_spans=` parameter to `token_ctor.Word()` above